### DISCUSSION TITLE
evaluating nutrient leaching

### DISCUSSION THREAD

**Discussion by CHARLES3241-SUDO** [APSIMInitiative/ApsimX#9290] [Q&A]
Hello all. am a new user of APSIM. is it possible to use this model to evaluate the amount of nutrient loss at different depths given that I have two factors i.e.. different fertilizer application rates and different irrigation scenarios? Kindly advise me. Thank you.

**Comment by BrianCollinss** [comment]
Hey @CHARLES3241-SUDO. Can you elaborate on "at different depths"?

**Comment by CHARLES3241-SUDO** [comment]
I mean, I want to evaluate nutrient leaching in sugarcane farm at a depth
of 45cm and 60cm for 3 different fertilizer application rates. In addition
I have another factor i.e 3 irrigation scenarios. Please guide me on how to
go about this. Thank you.

**Comment by BrianCollinss** [comment]
Look at these 'FlowX' state variables:

![image](https://github.com/user-attachments/assets/e6a4479f-582a-46c4-8ef4-41a9687684c3)

As for the factors, you need to put your simulation under an Experiment and use Factors. More info: https://apsimnextgeneration.netlify.app/usage/factorials/

**Comment by CHARLES3241-SUDO** [comment]
Thanks, I'll check it out.
